spyder_kernels/customize/tests/test_utils.py

```diff
@@ -16,7 +16,7 @@ def test_user_sitepackages_in_pathlist():
     if sys.platform.startswith('linux'):
         user_path = 'local'
     elif sys.platform == 'darwin':
-        user_path = '/Users/travis/.local'
+        user_path = '/Users/runner/.local'
```

Comment:
maybe 

`os.path.expanduser('~/.local')` is more portable ? it should also work locally

Thoughts @ccordoba12 ?